gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -1381,11 +1374,9 @@ private TypeAdapter<T> delegate() {
 
   @Override
   public String toString() {
-    return new StringBuilder("{serializeNulls:")
-        .append(serializeNulls)
-        .append(",factories:").append(factories)
-        .append(",instanceCreators:").append(constructorConstructor)
-        .append("}")
-        .toString();
+    return "{serializeNulls:" + serializeNulls +
+            ",factories:" + factories +
```

Comment:
Not sure if the indentation here is correct, I think this should have 4 spaces relative from the start of `return` (instead of 8).